src/index.test.js

```diff
@@ -1,5 +1,3 @@
-/* eslint ava/no-only-test:0, ava/no-skip-test:0 */
-import test from 'ava'
```

Comment:
this'll probably have to come from an external contribution but I'm happy to help anyone who wants to work on anything in this space :)